spec/requests/api/connect/v3/systems/products_controller_spec.rb

```diff
@@ -214,12 +230,21 @@
         ).to_json
       end
 
-      its(:code) { is_expected.to eq('201') }
-      its(:body) { is_expected.to eq(serialized_json) }
-      it 'updates the activation' do
-        activation = Activation.find(activation_id)
+      describe 'response' do
+        before { request }
+        subject { response }
+
+        its(:code) { is_expected.to eq('201') }
```

Comment:
This may be nitpicky, but HTTP 201 is for a created new resource. Return 204 instead?

Note:
This is not a blocking or an important issue for me.

Reply:
Hmm this is the HTTP code that is returned by glue, so RMT is just adhering to the glue API.

Reply:
No problem then